Composer/packages/client/src/recoilModel/dispatchers/storage.ts

```diff
@@ -165,6 +166,16 @@ export const storageDispatcher = () => {
       const data = response?.data;
 
       if (data && Array.isArray(data) && data.length > 0) {
+        (data as BotTemplate[]).push({
+          id: QnABotTemplateId,
+          name: 'generator-qna-bot',
+          description: 'Empty bot template that routes to qna configuration',
+          package: {
+            packageName: 'generator-empty-bot',
+            packageSource: 'npm',
+            packageVersion: '0.0.1',
```

Comment:
should these values be driven from the server response rather than the client?

Reply:
Good point, I have moved it to the server side